xarray/coding/cftimeindex.py

```diff
@@ -268,29 +268,32 @@ def _partial_date_slice(self, resolution, parsed):
         >>> from cftime import DatetimeNoLeap
         >>> import pandas as pd
         >>> import xarray as xr
-        >>> da = xr.DataArray([1, 2],
-                              coords=[[DatetimeNoLeap(2001, 1, 1),
-                                       DatetimeNoLeap(2001, 2, 1)]],
-                              dims=['time'])
-        >>> da.sel(time='2001-01-01')
+        >>> da = xr.DataArray(
+        ...     [1, 2],
+        ...     coords=[[DatetimeNoLeap(2001, 1, 1), DatetimeNoLeap(2001, 2, 1),]],
```

Comment:
the trailing comma (`,]],`) at the end of the line happens with `black`, too: calling `black` on a file containing
```python
da = xr.DataArray(
    [1, 2],
    coords=[[
        DatetimeNoLeap(2001, 1, 1),
        DatetimeNoLeap(2001, 2, 1),
    ]],
    dims=["time"],
)
```
gives us
```python
da = xr.DataArray(
    [1, 2],
    coords=[[DatetimeNoLeap(2001, 1, 1), DatetimeNoLeap(2001, 2, 1),]],
    dims=["time"],
)
```

while having
```python
da = xr.DataArray(
    [1, 2],
    coords=[[
        DatetimeNoLeap(2001, 1, 1),
        DatetimeNoLeap(2001, 2, 1)
    ]],
    dims=["time"],
)
```
correctly returns
```python
da = xr.DataArray(
    [1, 2],
    coords=[[DatetimeNoLeap(2001, 1, 1), DatetimeNoLeap(2001, 2, 1)]],
    dims=["time"],
)
```


I'm not sure why, though.